### DISCUSSION TITLE
Rewards for the top bug reporters January 2022

### DISCUSSION THREAD

**Discussion by Valsery** [chromiecraft/chromiecraft#2912] [General]
This is the number of VALID bug reports per every contributor that have been validated in January 2022.
When a bug is validated, it gets the Linked [AC] label.

If your bug hasn't been validated yet, don't worry: it will be counted in a later month. Invalid bug reports will of course not be counted.

Source: https://www.chromiecraft.com/top-contributors-of-january-2022/

The people mentioned here: in order to receive your Chromie points, please send me a private message on Discord (Gn#6416) with the following:

    1. your GitHub username
    2. your game account username
    3. your game account email

Remember to give us a Star on our main GitHub repo: https://github.com/azerothcore/azerothcore-wotlk

<a href=https://github.com/ishidinho>@ishidinho</a> - 10
<a href=https://github.com/Elexias>@Elexias</a> - 6
<a href=https://github.com/j-hdt>@j-hdt</a> - 5
<a href=https://github.com/Sathrenor>@Sathrenor</a> - 5
<a href=https://github.com/Scapulla>@Scapulla</a> - 3
<a href=https://github.com/Hyperbearr>@Hyperbearr</a> - 3
<a href=https://github.com/Clusterfonk>@Clusterfonk</a> - 3
<a href=https://github.com/Apachemoo>@Apachemoo</a> - 3
<a href=https://github.com/zranitel>@zranitel</a> - 2
<a href=https://github.com/xChaosityx>@xChaosityx</a> - 2
<a href=https://github.com/luckdemon>@luckdemon</a> - 2
<a href=https://github.com/brookie2664>@brookie2664</a> - 2
<a href=https://github.com/baranzadeoglu>@baranzadeoglu</a> - 2
<a href=https://github.com/Vladimirjov>@Vladimirjov</a> - 2
<a href=https://github.com/Swindellite>@Swindellite</a> - 2
<a href=https://github.com/Santaria94>@Santaria94</a> - 2
<a href=https://github.com/Saikur0n>@Saikur0n</a> - 2
<a href=https://github.com/KCorruption>@KCorruption</a> - 2
<a href=https://github.com/HasieEST>@HasieEST</a> - 2
<a href=https://github.com/FTGCocolandia>@FTGCocolandia</a> - 2
<a href=https://github.com/Clanggedon7777>@Clanggedon7777</a> - 2
<a href=https://github.com/warmesTghosT>@warmesTghosT</a> - 1
<a href=https://github.com/temperrr>@temperrr</a> - 1
<a href=https://github.com/ohkasdasf>@ohkasdasf</a> - 1
<a href=https://github.com/mrzywiec>@mrzywiec</a> - 1
<a href=https://github.com/mcraig000>@mcraig000</a> - 1
<a href=https://github.com/matisskeiris>@matisskeiris</a> - 1
<a href=https://github.com/mandris36>@mandris36</a> - 1
<a href=https://github.com/lovemelovely>@lovemelovely</a> - 1
<a href=https://github.com/jukefr>@jukefr</a> - 1
<a href=https://github.com/jeskej>@jeskej</a> - 1
<a href=https://github.com/immersionnn>@immersionnn</a> - 1
<a href=https://github.com/godnesso>@godnesso</a> - 1
<a href=https://github.com/formiclol>@formiclol</a> - 1
<a href=https://github.com/esliwka>@esliwka</a> - 1
<a href=https://github.com/dennots>@dennots</a> - 1
<a href=https://github.com/claudiodfc>@claudiodfc</a> - 1
<a href=https://github.com/chaossonic>@chaossonic</a> - 1
<a href=https://github.com/andyxtoday>@andyxtoday</a> - 1
<a href=https://github.com/Zosina>@Zosina</a> - 1
<a href=https://github.com/YunataSavior>@YunataSavior</a> - 1
<a href=https://github.com/Soulex23>@Soulex23</a> - 1
<a href=https://github.com/Scrublettt>@Scrublettt</a> - 1
<a href=https://github.com/Renatuh95>@Renatuh95</a> - 1
<a href=https://github.com/Reily93>@Reily93</a> - 1
<a href=https://github.com/Porchigone16>@Porchigone16</a> - 1
<a href=https://github.com/Plan414>@Plan414</a> - 1
<a href=https://github.com/Nyeriah>@Nyeriah</a> - 1
<a href=https://github.com/Never6>@Never6</a> - 1
<a href=https://github.com/Moilert>@Moilert</a> - 1
<a href=https://github.com/Mikel0ke>@Mikel0ke</a> - 1
<a href=https://github.com/Knascher>@Knascher</a> - 1
<a href=https://github.com/JonDamodon>@JonDamodon</a> - 1
<a href=https://github.com/Hihilist>@Hihilist</a> - 1
<a href=https://github.com/Fabbsie>@Fabbsie</a> - 1
<a href=https://github.com/DJFoxtrot>@DJFoxtrot</a> - 1
<a href=https://github.com/CMHWoody>@CMHWoody</a> - 1
<a href=https://github.com/Amber-Haze>@Amber-Haze</a> - 1
<a href=https://github.com/15414h>@15414h</a> - 1

**Comment by balleny** [comment]
Shin used to send out an email notification, not sure people in this list got an notification tho.
